docs/source/reference/pysupported.rst

```diff
@@ -229,6 +229,52 @@ The following functions from the :mod:`operator` module are supported:
 * :func:`operator.truediv`
 * :func:`operator.xor`
 
+.. _pysupported-random:
+
+``random``
+----------
+
+Numba supports top-level functions from the :mod:`random` module, but does
+not allow you to create individual Random instances.  A Mersenne-Twister
+generator is used, with a dedicated internal state.  It is initialized at
+startup with entropy drawn from the operating system.
+
+* :func:`random.betavariate`
+* :func:`random.expovariate`
+* :func:`random.gammavariate`
+* :func:`random.gauss`
+* :func:`random.getrandbits`: number of bits must not be greater than 64
```

Comment:
Is it possible do give a warning/error to the user when the number of bits is greater than 64? It seems that if the number of bits is greater than 64, the output begins to diverge from CPython's without reporting any error - it appears that could be easy to miss this warning in the docs and not notice what is happening.